webui/react/src/components/kit/InlineForm.tsx

```diff
@@ -0,0 +1,87 @@
+import { Form } from 'antd';
+import React, { useCallback, useState } from 'react';
+
+import Button from 'components/kit/Button';
+import Icon from 'components/kit/Icon';
+
+import css from './InlineForm.module.scss';
+
+interface Props {
+  label: string;
+  inputValue?: string | number;
+  onSubmit: (inputValue: string | number) => Promise<void | Error> | void;
+  inputElement: React.ReactNode;
+  required?: boolean;
+  forceEdit?: boolean;
+}
+
+const InlineForm: React.FC<Props> = ({
+  label,
+  required,
+  forceEdit = false,
+  inputElement,
+  inputValue = '',
```

Comment:
This should just be `initialValue`.

Reply:
Also it would be great if you didn't conflate initialValue and `value` and allowed the component to be uncontrolled.

Reply:
`initialValue` and `value` have different uses. You should preserve the API that a user would get from using `Form` and `Form.Item` together. As far as I can see you just changed the component from only allowing controlled to now only allowing uncontrolled.